README.md

```diff
@@ -11,15 +30,47 @@ To run it locally, run
 ```
 npm run start
 ```
+#### Environment Variables
+If the home page is blank, check that there is a file named `.env.local` in the root of the repo. 
+If the file does not exist, please go to mapbox and get an _access token of your own_, put
+
+```
+REACT_APP_MAPBOX_ACCESS_TOKEN=<YOUR ACCESS TOKEN>
+```
+
+inside that `.env.local`. Then re-run `npm run start`
+
+<br></br>
+To run all the test cases, enter
+```
+npm run test
+```
+To see code coverage, enter
+```
+npm run test -- --coverage
```

Comment:
coverage testing is a little lost in this environment section - not a big deal but could make it its own section or change the title to include testing

Reply:
@abentler Addressed with  commit 3235c76dd7f93b2143ea641c34048feaad397cb0